CRT.c

```diff
@@ -1046,7 +1086,9 @@ void CRT_init(const Settings* settings, bool allowUnicode, bool retainScreenOnEx
       CRT_scrollHAmount = 5;
    }
 
-   if (termType && (String_startsWith(termType, "xterm") || String_eq(termType, "vt220"))) {
+   /*check termTypes to determine if term supports escape sequences for defined
+    * keys */
```

Comment:
Keep this on one line.
```suggestion
   /* check termTypes to determine if term supports escape sequences for defined keys */
```

Reply:
I think this comment is outdated and we may just remove it. I suggested adding that comment before the function `termSupportsDefinedKeys` is added so that people can know what the conditional is about. Now the `termSupportsDefinedKeys` name is self-documenting that we can remove this comment line.